.github/workflows/release.yml

```diff
@@ -0,0 +1,100 @@
+name: Release
+on:
+  push:
+    tags: '*'
+
+# Needed to create the release
+permissions:
+  contents: write
+  id-token: write
+
+# Needed to create the release and use gh cli
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN:  ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout Code
+          uses: actions/checkout@v4 
+        - name: Create a Release
+          id: create_release
+          uses: actions/create-release@v1
+          with:
+            tag_name: ${{ github.ref }}
+            release_name: Release ${{ github.ref_name }}
+            prerelease: true
+
+  build-and-push-amd64:
+    runs-on: ubuntu-latest
+    needs: [create-release]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+      - name: Set up GO
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.20'
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Run Dapper CI
```

Comment:
I don't think we need to run Dapper CI on release, only on `pull_request` I think.

Reply:
Workflows and runs won't share state or data between them. So we have to build from again for releases.

Reply:
In that case, I think you just need `make build`.